src/web/utils.ts

```diff
@@ -64,15 +64,18 @@ export function tryExtractStylusData(
     return;
   }
 
+  // @ts-ignore This property exists (https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent#instance_properties)
+  const eventAzimuthAngle = event.azimuthAngle;
+  // @ts-ignore This property exists (https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent#instance_properties)
+  const eventAltitudeAngle = event.altitudeAngle;
```

Comment:
I think those will be typed as `any`, if so explicitly setting them to `number | undefined` may be a good idea.

Reply:
I agree, done in [18d137f](https://github.com/software-mansion/react-native-gesture-handler/pull/3107/commits/18d137f8fa9cb0e9e3c9e6edace10f314dceb12d)